lib/runner/cli/nightwatch.conf.ejs

```diff
@@ -253,6 +253,61 @@ module.exports = {
       }
     },
     //////////////////////////////////////////////////////////////////////////////////
+    // Configuration for when using the SauceLabs cloud service                      |
+    //                                                                               |
+    // Please set the username and access key by setting the environment variables:  |
+    // - SAUCE_USERNAME                                                              |
+    // - SAUCE_ACCESS_KEY                                                            |
+    //////////////////////////////////////////////////////////////////////////////////
+    sauce: {
```

Comment:
can you call it `saucelabs`?

Reply:
I was trying to keep it in their naming convention where "sauce" is used often, but if you prefer I can change it to "saucelabs". Please let me know if you'd still like it changed.

Reply:
yeah, I think saucelabs is a better fit.

Reply:
Would it be weird then that we name the section as `saucelabs` but then use the shorthand `sauce:options` in the lower section? They expect their settings to be in `sauce:options` specifically https://docs.saucelabs.com/dev/test-configuration-options/#w3c-webdriver-capabilities--required

Reply:
Renamed to saucelabs